src/components/Button/index.js

```diff
@@ -1,22 +1,49 @@
-import React, {PropTypes} from 'react'
-
-const Button = ({
-  children, 
-  onClick, 
-  className,
-}) => (
-  <button
-    className={`button-reset bn dib b0 ttu dim br2 ph3 pv2 white bg-orange tc ${className}`}
-    onClick={onClick}
-  >
-    {children}
-  </button>
-)
+import React, { PropTypes } from 'react'
+
+const commonClasses = 'link dib f5 fw6 tracked tc br2 ttu ba pointer'
+
+const sizes = ['large', 'small', 'xl']
+const types = ['success', 'warning', 'danger', 'primary', 'default']
+
+const sizedBtnClasses = {
+  large: 'h3 pa3 eh-btn-primary',
+  small: 'h2 lh-solid pa3 eh-btn',
+  xl: 'lh-solid ph4 pv4 eh-xl'
```

Comment:
if I was being pedantic I'd say `extra-large` since we aren't abbreviating the others. ;)